support/src/main/java/org/cardanofoundation/lob/app/support/spring_web/SpringWebConfig.java

```diff
@@ -19,6 +25,11 @@ public class SpringWebConfig {
     @Value("${lob.cors.allowed.origins:http://localhost:3000}")
```

Comment:
@Kammerlo is this url suppose to be dev url? 'https://dev.cf-lob.metadata.dev.cf-deployments.org' currently endpoints with authorization not working. trying to see we missed anything.